test/doc/math/pow.cpp

```diff
@@ -1,18 +1,19 @@
 // revision 1
 #include <eve/module/math.hpp>
 #include <iostream>
+#include <cmath>
 
 int main()
 {
-  eve::wide pf = {2.0f, 3.0f, -4.0f, 2.0f, 2.0f,
+  eve::wide qf = {0.0f, 3.0f, -4.0f, 2.0f, 2.0f,
                   eve::inf(eve::as<float>()), eve::minf(eve::as<float>()), eve::nan(eve::as<float>())};
-  eve::wide qf = {4.0f, 1.0f, -1.0f,  0.5f,  0.0f, 2.0f, -3.0f,  2.5f};
+  eve::wide pf = {4.0f, 1.0f, -1.0f,  0.5f,  0.0f, 0.0f, -0.0f,  2.5f};
```

Comment:
why changing those values ?

Reply:
I had an error due to is_minf and this made it visible